### ISSUE TITLE
issues about the test set

### ISSUE THREAD

**Issue by WERush** [fangchangma/sparse-to-dense#6]
Hi,
I have several questions about the testing
I want to test your trained model with raw images and depth. How to make it work? 
or do you have a HDF5 file for testing only?

Thanks very much.

**Comment by fangchangma** [comment]
I just uploaded the validation dataset, in case you don't want to download the entire training dataset. The validation dataset is available at http://datasets.lids.mit.edu/sparse-to-dense/data/nyudepthv2_val_official.tar.gz.

As a side note, be mindful when using "raw" color images and depth images, since they are not registered (the cameras are not aligned). All the training and validation data here are already registered into the rgb frame.
